client/src/client_sync/v20/generating.rs

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! Macros for implementing JSON-RPC methods on a client.
+//!
+//! Specifically this is methods found under the `== Generating ==` section of the
+//! API docs of Bitcoin Core `v0.20`.
+//!
+//! All macros require `Client` to be in scope.
+//!
+//! See or use the `define_jsonrpc_minreq_client!` macro to define a `Client`.
+
+/// Implements Bitcoin Core JSON-RPC API method `generatetodescriptor`
+#[macro_export]
+macro_rules! impl_client_v17__generate_to_descriptor {
```

Comment:
In 38b6fe9d305f345593b4dcd2f115320161d2170b

cut'n'pasta error here mate, should be v20